percona_pg_telemetry.c

```diff
@@ -668,6 +668,52 @@ write_pg_settings(void)
 	CommitTransactionCommand();
 }
 
+static void
+write_tde_info(List *kmslist)
+{
+	char		buf[4096] = {0};
+	size_t		buf_size = sizeof(buf);
+	FILE	   *fp;
+	int			flags;
+	ListCell   *lc;
+	char		str[2048] = {0};
```

Comment:
These two constant lenght feel pretty arbitrary. Where do they come from?

Reply:
I also wonder if we long term should not move to using `StringInfo` when building strings. That is the idiomatic way in PostgreSQL. But that is a question for a separate commit.